carla/data/catalog/catalog.py

```diff
@@ -1,11 +1,12 @@
 import pandas as pd
 import yaml
 
-from . import processing
+from ..api import Data
+from ..helper import processing
```

Comment:
There is this programming notion that util/helper/common packages are bad design and should be named according to there usecase. Should we instead name it to `processing/processing.py` (if you want to nest the file like the others)

- https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common
- https://www.adam-bien.com/roller/abien/entry/util_packages_are_evil

Reply:
Sounds good, will implement this later on